package.json

```diff
@@ -52,9 +52,9 @@
     "test.watch": "jest --watch",
     "test.watch-all": "jest --watchAll --coverage",
     "tsc": "tsc --incremental",
-    "tsc.prod": "tsc",
+    "tsc.prod": "tsc --build --force",
     "tsc.scripts": "tsc -p scripts/tsconfig.json",
-    "tsc.watch": "tsc --incremental --watch"
+    "tsc.watch": "tsc --incremental --watch --build"
```

Comment:
In order to build with project references we need to use the `--build` flag. If you want to check out what it's doing you can also pass `--verbose` and it will show you want its building / rebuilding and why.